jellyfin-api/src/main/kotlin-generated/org/jellyfin/sdk/api/operations/MediaInfoApi.kt

```diff
@@ -86,7 +86,7 @@ public class MediaInfoApi(
 
 	/**
 	 * For backwards compatibility parameters can be sent via Query or Body, with Query having higher
-	 * precedence.
+	 * precedence.
```

Comment:
I have absolutely no idea why this keeps happening, I guess it's a Windows<>Linux thing?

Reply:
Tabs vs spaces maybe?

Reply:
nop, no whitespace changes

Reply:
Perhaps the CRLF character has changed?

Reply:
But why wouldn't that happen in other files? All of them are regenerated

Reply:
> Perhaps the CRLF character has changed?

That in fact seems to be the case - CRLF (Windows) gets changed to LF (UNIX).

Git diff is
`09 20 2a 20 70 72 65 63 65 64 65 6e 63 65 2e 0d 0a`
vs
`09 20 2a 20 70 72 65 63 65 64 65 6e 63 65 2e 0a`